37/index.html

```diff
@@ -436,7 +436,7 @@
 </head>
 <body id="index" class="article toc2 toc-left">
 <div id="header">
-<h1>WildFly Documentation</h1>
+<h1>WildFly 37 Documentation</h1>
```

Comment:
For this we'll need an update to https://raw.githubusercontent.com/wildfly/wildfly/refs/heads/main/docs/src/main/asciidoc/index.adoc as well -- that's the file used to generate this page for future releases.

Reply:
I've noticed you have not pushed the requested change yet. Look at the first line of [index.adoc](https://github.com/wildfly/wildfly.github.io/pull/97/files#diff-bae68ca21748a8f2d4b9d7e2530baee437cd4d1f799e0a5e705613571be386a8), it should be:

```
= WildFly {wildfly-latest-major} Documentation
```

Reply:
@bstansberry I guess you are talking about the first line on the index.adoc? could you confirm that what  you want to see is the WildFly version on the main title of the WildFly Documentation landing page?

Otherwise, the 37/index.html is generated from the index.adoc file at WildFly repo: https://github.com/wildfly/wildfly/blob/main/docs/src/main/asciidoc/index.adoc